src/authentication/renderer/AuthenticationApp.vue

```diff
@@ -77,6 +77,7 @@ import NcTextField from '@nextcloud/vue/dist/Components/NcTextField.js'
 import { getCapabilities, getCurrentUserData } from '../../shared/ocs.service.js'
```

Comment:
'getCurrentUserData' is defined but never used

```suggestion
import { getCapabilities } from '../../shared/ocs.service.js'
```